config/areas/system/views.php

```diff
@@ -21,14 +21,15 @@
 			// @codeCoverageIgnoreEnd
 
 			$plugins = $system->plugins()->values(function ($plugin) {
+				$authors = $plugin->authorsNames();
 				return [
-					'author'  => $plugin->authorsNames(),
-					'license' => $plugin->license(),
+					'author'  => empty($authors) ? '–' : $authors,
+					'license' => $plugin->license() ?? '–',
 					'name'    => [
-						'text' => $plugin->name(),
-						'href' => $plugin->link(),
+						'text' => $plugin->name() ?? '–',
+						'href' => $plugin->link() ?? '–',
```

Comment:
A dash as the link href isn't useful. I think we need to keep passing `null` here and then the frontend should render text instead of a link (if it doesn't already do that).